dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/request/IRequestImpl.java

```diff
@@ -70,6 +70,8 @@ public class IRequestImpl implements IRequestExtended
   private Boolean isSSL;
   private static boolean normalizeRequestURI = WCCustomProperties.NORMALIZE_REQUEST_URI; //PI05525
 
+  private static final HttpHeaderKeys CONTENT_TYPE_KEY = HttpHeaderKeys.HDR_CONTENT_TYPE;
```

Comment:
Not sure why making another static final for this instead of using what it refers to directly.

Reply:
YES!  I had that thought exactly!! and then forgot to change it ... sigh  :-))
will do so, and for the others as well